### DISCUSSION TITLE
Error when starting docker: "cannot load certificate key"

### DISCUSSION THREAD

**Discussion by amandavrVitat** [ptrumpis/snap-camera-server#73] [Q&A] [answered]
When starting docker, one of the "webproxy-1" containers does not start and gives the following error:

`nginx: [emerg] cannot load certificate key "/etc/ssl/private/studio-app.snapchat.com.key": BIO_new_file() failed (SSL: error:80000001:system library::Operation not permitted:calling fopen(/etc/ssl/private/studio-app.snapchat.com.key, r) error:10080002:BIO routines::system lib)`

I've already tried changing the docker file:
 ```
webproxy:
    image: nginx:1:22
```
but it did not work.

The certificate permissions are also all set to "always allow".

Any suggestions on how to resolve it?

**Comment by ptrumpis** [comment, marked as answer]
Assuming your server files are located at `/Applications/snap-camera-server-3.1.1/`

Run the Terminal Command
```sh
sudo chown -R $(id -un):$(id -gn) /Applications/snap-camera-server-3.1.1/ssl/*
```

This should fix the problem

**Reply by amandavrVitat** [reply]
It worked. Thanks!

**Comment by radmasa** [comment]
Note, I was having the following error:
error while creating mount source path /host_mnt/Applications/snap-camera-server-3.2.1/ssl/studio-app.snapchat.com.key': mkdir
/host_mnt/Applications/snap-camera-server-3.2.1/ssl/studio-app.snapchat.com.key: permission denied

This same command (updated for version 3.2.1) corrected the situation.
`sudo chown -R $(id -un):$(id -gn) /Applications/snap-camera-server-3.2.1/ssl/*`
